roles/oneagent/tests/conmponent/scripts/util/test_helpers.py

```diff
@@ -0,0 +1,97 @@
+import functools
+import logging
+from pathlib import Path
+from typing import Callable, Any
+
+from command.platform_command_wrapper import PlatformCommandWrapper
+
+from technology.deployment_config import DeploymentConfig
+from technology.deployment_runner import DeploymentRunner
+from util.common_utils import get_oneagentctl_path, get_platform_argument
+from util.constants.common_constants import INSTALLER_PARTIAL_NAME, HOST_SERVER_ADDRESS, HOST_SERVER_TOKEN
+from util.test_data_types import DeploymentPlatform, DeploymentResult, PlatformCollection
+
+CallableOperation = Callable[[DeploymentPlatform, str, Any], None]
+
+
+def _get_param_by_name(name: str, **kwargs) -> Any:
+    assert name in kwargs, f"No '{name}' parameter in parameters list"
+    return kwargs[name]
+
+
+def disable_for_localhost():
```

Comment:
```suggestion
def skip_hosts_for_tests(host_values: list[str]):
```

Reply:
consider a different approach as this function wouldn't work for addresses such as `127.0.0.1`. why is skipping tests needed for localhost?